### DISCUSSION TITLE
Please support SMTP

### DISCUSSION THREAD

**Discussion by ryantxr** [msveshnikov/allchat#40] [Ideas]
I was looking through the code and I noticed this:
```
const transporter = nodemailer.createTransport({
    service: "gmail",
    auth: {
        user: process.env.EMAIL,
        pass: process.env.EMAIL_PASSWORD,
    },
});
```
Seems like this should support SMTP. I'm not really inclined to put my gmail password in here.

**Comment by msveshnikov** [comment]
It supports both SMTP for sending you some information (if you asked) and IMAP for reading inbox and answering your inquires via email (you have to be registered for this). So here you put your assistant credentials, not yours

**Comment by msveshnikov** [comment]
But if you want just to send emails to custom SMTP (not Gmail) it is one liner change. So please fork and adjust
